mon-pix/app/styles/pages/_join-restricted-campaign.scss

```
@@ -22,12 +22,14 @@
     }
   }
 
-  &__container {
-    padding-bottom: 30px;
+  &__back-button {
+    padding: 8px;
+    text-align: left;
+  }
 
-    @include device-is('tablet') {
-      padding: 40px;
-    }
+  &__container {
```

Créer une class spécifique pour centrer le bouton "C'est parti"
```suggestion
  &__form-wrapper {
```